docs/maps/search.asciidoc

```diff
@@ -0,0 +1,37 @@
+[role="xpack"]
+[[maps-search]]
+== Searching Your Data
+
+The **Maps** application embeds the query bar with autocomplete for real-time ad hoc search.
+All layers with {es} sources are updated when you submit a search request.
+
+image::maps/images/global_search_bar.png[]
+
+
+[float]
+=== Layer with {es} source
```

Comment:
This feels like it belongs in a separate doc with descriptions of when to use each.  I'm also not sure how the content about layers here relates to the sections about the Heat map layer, Tile layer, and Vector layer.

If you want to include information about layers here, provide more context, especially in relation to search. Are these the only types of layers you can search?

Reply:
> If you want to include information about layers here, provide more context, especially in relation to search. Are these the only types of layers you can search?

Yes, this section is defining which layers will be effected by the global search bar

Reply:
Maybe you can add some wording around that.